### PR TITLE
[stable23] Add testing for mariadb10.6

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

**Comment by szaimen** [comment]
needs a rebase after https://github.com/nextcloud/docker-ci/actions

**Comment by szaimen** [comment]
Seems like somehow is this package private and I am not able to change its visibility: https://github.com/orgs/nextcloud/packages/container/continuous-integration-mariadb-10.6/settings

File: .drone.yml [hunk 1]
```diff
@@ -290,14 +290,14 @@ trigger:
 
 ---
 kind: pipeline
-name: mariadb10.1-php7.3
```
Comment (nickvergessen) [inline .drone.yml:293]: We can not drop an old version in a maintenance release. We have to continue supporting and therefore testing what was supported when it came out. We can add things on top, no issue.

**Review by nickvergessen — DISMISSED** [review]

File: .drone.yml (hunk 1 above)
Comment (szaimen) [reply .drone.yml:293]: all right! Last time I did this you told me to not add things on top but will do this then 👍

Comment (nickvergessen) [reply .drone.yml:293]: Well it's a difference between master and stable*

Comment (szaimen) [reply .drone.yml:293]: okay, good to know!

Comment (szaimen) [reply .drone.yml]: done

**Review by blizzz — APPROVED** [review]

**Comment by szaimen** [comment]
failing CI unrelated

**Pull request merged**